homeassistant/components/thread/discovery.py

```diff
@@ -59,15 +61,30 @@ def try_decode(value: bytes | None) -> str | None:
     server = service.server
     vendor_name = try_decode(service.properties.get(b"vn"))
     thread_version = try_decode(service.properties.get(b"tv"))
+    unconfigured = None
```

Comment:
should this be False ?

Reply:
I'm not sure. My idea was to set the unconfigured flag to `True` in some cases where we know the router is not correctly configured.
In most cases we don't know however, that's why I let `None` be the default instead of `False`.